test/versioned/kafkajs/kafka.tap.js

```diff
@@ -195,6 +197,8 @@ tap.test('sendBatch records correctly', (t) => {
         eachMessage: async ({ message: actualMessage }) => {
           t.equal(actualMessage.value.toString(), message)
           t.match(actualMessage.headers['x-foo'].toString(), 'foo')
+          t.equal(actualMessage.headers.newrelic.toString(), '')
```

Comment:
these are fine but it's only asserting 1 message. you could add this same assertion to `consume inside of a transaction` which is sending multiple messages